src/components/ProductListContainer/styles.module.scss

```diff
@@ -0,0 +1,14 @@
+.cards {
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  margin-bottom: 80px;
+}
+.productCard {
+  flex: 0 0 auto;
+}
+
+.cards::-webkit-scrollbar {
+  display: none;
+}
```

Comment:
I like the approach but this has a problem on different browsers.

If the browser is not webkit based, then the scrollbar is present (because it doesn't enter the `display: none` rule).
If the browser is webkit based, then you can't see the rest of the cards in desktop because the scrollbar doesn't appear (and you can't `touch` in desktop)

We should aim for the same usability for different environments. I suggest we remove the `webkit-overflow-scrolling` (it's also non-standard and only webkit oriented) and the `-webkit-scrollbar` rule (so there always exist a scrollbar to use and see the products.

The scrollbar can be ugly to see, but the content must always be accessible. One alternative is to change the wrapping to `flex-wrap: wrap` (the content is there, only that is presented a bit differently).